docs/support/index.md

```diff
@@ -0,0 +1,25 @@
+# Getting Help
+
+This page describes how you can get help with publishing or using OFDS data.
+
+## OFDS Helpdesk
```

Comment:
I prefer to avoid 'helpdesk' terminology as many people read it to mean resetting passwords etc. :-)

```suggestion
## OFDS Support Team
```